app/serializers/api/v1/ecf_induction_record_serializer.rb

```diff
@@ -0,0 +1,76 @@
+# frozen_string_literal: true
+
+require "jsonapi/serializer/instrumentation"
+
+module Api
+  module V1
+    class ECFInductionRecordSerializer
+      include JSONAPI::Serializer
+      include JSONAPI::Serializer::Instrumentation
+
+      set_type :user
+
+      USER_TYPES = {
+        early_career_teacher: "early_career_teacher",
+        mentor: "mentor",
+        other: "other",
+      }.freeze
+
+      CIP_TYPES = {
+        ambition: "ambition",
+        edt: "edt",
+        teach_first: "teach_first",
+        ucl: "ucl",
+        none: "none",
+      }.freeze
+
+      set_id :id do |induction_record|
+        induction_record.preferred_identity.external_identifier
```

Comment:
we currently get the external id from the participant profile, however the email from the preferred identity